docker/utils/utils.py

```diff
@@ -511,6 +511,9 @@ def create_host_config(
     if privileged:
         host_config['Privileged'] = privileged
 
+    if oom_kill_disable:
```

Comment:
Can we test that `version_gte(version, '1.19') is True and raise an `InvalidVersion` otherwise?